SoftLayer/CLI/bandwidth/pools_create.py

```diff
@@ -21,13 +21,36 @@
     "FRA": "FRA"
 }
 
+regions = ['SJC/DAL/WDC/TOR/MON', 'AMS/LON/MAD/PAR', 'SNG/HKG/OSA/TOK', 'SYD', 'MEX', 'SAO', 'CHE', 'MIL', 'SEO', 'FRA']
+HELP_STMT = "Ex: 'HKG' or 'SNG/HKG/OSA/TOK'"
+
+
+def check_region_param(ctx, param, value):
+    """Check if provided region is region group or part of region"""
+
+    # :params string value: Region or Region-Groups
+    # return string Region-Groups
+
+    _ = [ctx, param]
+    region_group = None
+    for key in location_groups:
+        if value in key or value is key:
+            region_group = key
+        else:
+            continue
```

Comment:
Is this line needed? If not I think we can remove it.

Reply:
Removed and tested